content/pages/download.md

```diff
@@ -151,4 +151,39 @@ versions:
         - slug: source
           name: 2.2.4 release
           url: https://github.com/mixxxdj/mixxx/archive/release-2.2.4.tar.gz
+  testing:
+    title: Development Snapshots
+    text: |
+      Testing the latest code we're working on and giving early feedback is a great way to contribute to Mixxx. Refer to the [Testing wiki page](https://github.com/mixxxdj/mixxx/wiki/Testing) for where to find the latest builds and instructions how to test pull requests before they are merged.
+
+      **Development snapshots are not intended for live use!** Expect crashes and make sure to [back up your Mixxx settings and library](https://manual.mixxx.org/2.4/en/chapters/getting_started.html#upgrading-mixxx) before upgrading.
```

Comment:
I think we should add that explicit info (OS-specific paths) to the [Testing wiki page](https://github.com/mixxxdj/mixxx/wiki/Testing) so testers find the relevant one place. (no need for redundant links though IMO)

```suggestion
      **Development snapshots are not intended for live use!** Expect crashes and make sure to back up your Mixxx settings and library before upgrading as explained in the Testing wiki page.
```